src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
> When we move the menu (choice) to dosemu:
> Do we still want to have scripts to put a dos in the system-wide location?

All scripts should support some form of --dest
and put things anywhere. When user wants to just
install DOS from internet, the temporary location
can be somewhere in /tmp. But if he wants to set
up the system-wide installation source, he can manually
do so with this script (perhaps also using sudo).

> The detection of what dos is there is explicitly needed for the situation this is
> directory is used.

Why??
Currently dosemu looks for freedos in /usr/[local/]share/dosemu/freedos
It will not look for ms-dos exactly there, obviously. ms-dos
will be in /usr/[local/]share/dosemu/msdos. So dos detection
logic is never needed, just use the proper dir names.